checkov/terraform/checks/resource/oci/SecurityListUnrestrictedIngress22.py

```diff
@@ -0,0 +1,43 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class SecurityListUnrestrictedIngress22(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure VCN inbound security lists allow all traffic on SSH port."
+        id = "CKV_OCI_19"
+        supported_resources = ['oci_core_security_list']
+        categories = [CheckCategories.NETWORKING]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        if 'ingress_security_rules' in conf:
+            self.evaluated_keys = ['ingress_security_rules']
+            rules = conf.get("ingress_security_rules", [])
+
+            for idx, rule in enumerate(rules):
+                if "0.0.0.0/0" in rule['source'][0] \
+                        and not (
+                        (rule['protocol'][0] != '1' and ('udp_options' not in rule) and ('tcp_options' not in rule))
+                        or (self.scan_protocol_conf(rule, 'tcp_options', idx) != CheckResult.FAILED
+                            and self.scan_protocol_conf(rule, 'udp_options', idx) != CheckResult.FAILED)
+                        or rule['protocol'][0] == 'all'):
+                    self.evaluated_keys = [f'ingress_security_rules/[0]/[{idx}]']
+                    return CheckResult.FAILED
+
+            return CheckResult.PASSED
+
+        return CheckResult.FAILED
```

Comment:
@ayajbara So if there's no ingress rules, port 22 is exposed to the world? Is there some documentation on this?